### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by stubailo**
One page for now, and not totally finished.

Before: https://www.apollographql.com/docs/link/links/http.html
After: https://deploy-preview-336--apollo-link-docs.netlify.com/docs/link/links/http.html

### CODE REVIEW

File: packages/apollo-link-http/README.md [hunk 1]
```diff
@@ -1,54 +1,49 @@
 ---
-title: Http Link
+title: apollo-link-http
```
Comment (peggyrayzis) [inline packages/apollo-link-http/README.md]: Are you going to change all the other links to have their package name as the title? Right now, it looks inconsistent on the deploy preview.

File: docs/source/index.md [hunk 2]
```diff
@@ -1,27 +1,26 @@
 ---
-title: Apollo Link
+title: Composable networking for GraphQL
 sidebar_title: Introduction
 description: Apollo Link is a standard interface for modifying control flow of GraphQL requests and fetching GraphQL results.
 ---
 
-This is the official guide for getting started with Apollo Link in your application. Apollo Link is a simple yet powerful way to describe how you want to execute a GraphQL operation, and what you want to do with the results. It is completely customizable, which means you can use links with Apollo Client, `graphql-tools`, GraphiQL, and even as a standalone client.
+This is the official guide for getting started with Apollo Link in your application. Apollo Link is a simple yet powerful way to describe how you want to get the result of a GraphQL operation, and what you want to do with the results. You can use Apollo Link with Apollo Client, `graphql-tools` schema stitching, GraphiQL, and even as a standalone client, allowing you to reuse the same authorization, error handling, and control flow across all of your GraphQL fetching.
```
Comment (peggyrayzis) [inline docs/source/index.md]: So much better!!

File: packages/apollo-link-http/README.md (hunk 1 above)
Comment (stubailo) [reply packages/apollo-link-http/README.md]: Yeah, just wondering if you think it's a good idea!

Comment (peggyrayzis) [reply packages/apollo-link-http/README.md]: I like it better with the package name :)

Comment (stubailo) [reply packages/apollo-link-http/README.md]: Dope. will do the rest

**Pull request merged**